cmd/stellar-rpc/internal/methods/get_ledger_entries.go

```diff
@@ -146,3 +201,82 @@ func NewGetLedgerEntriesHandler(logger *log.Entry, ledgerEntryReader db.LedgerEn
 		return response, nil
 	})
 }
+
+type keyEntriesAndOrdering struct {
```

Comment:
Is this necessary because the query server doesn't return the keys in request order?

Reply:
Yep, in fact, I don’t think there are any guarantees on the DB either.

Reply:
Oof, subtle. Nice job :)